docs/milestone-1-validation.md

```diff
@@ -0,0 +1,99 @@
+# Milestone-1 Validation
+
+* Scality Object Storage Interoperability Service (for short, **Scality OSIS**)
+* VMware Cloud Director Object Storage Extension (for short, **vCloud Director OSE**)
+
+## Prerequisites
+
+1. Run Vault (check [How to Run Vault](https://github.com/scality/Vault/blob/development/7.10/TESTING.md#how-to-run-1))
+   
+1. Run Cloudserver (check [running Cloudserver here](https://github.com/scality/cloudserver/tree/development/7.10#installation)) with env variable: `S3VAULT=scality` so that it can use vault as auth backend.
```

Comment:
I would not link to the 7.10 version specifically, especially considering the PR I created that includes that env var is not even pushed. Preferably we would just link to the generic docs page but maybe @wabernatScality will have another idea.

Reply:
I realize now I kinda contradicted myself by saying not to use direct file links as they are not stable while also saying this file feels more like a temporary thing than permanent...
So I guess ignore that if you want but something to potentially consider. :smirk:

Reply:
As this is is only for milestone-1 and milestone-1 should ship with 7.10., this should be good